### ISSUE TITLE
repo_reuse_counts test fails

### ISSUE THREAD

**Issue by hlapp** [NESCent/1000-datasets#2]
This is from a branch on which I removed the commit in #1, so can't have anything to do with that.

```
* testing repo_reuse_counts **
Traceback (most recent call last):
  File "run_tests.py", line 13, in <module>
    mod = importlib.import_module('scripts.' + script)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/lapp/Work/1000-datasets/scripts/repo_reuse_counts.py", line 22, in <module>
    n = int(line[:8])
ValueError: invalid literal for int() with base 10: '1 \t\tB'
```

**Comment by bendmorris** [comment]
It looks like you're using a Mac? This is probably due to differences in `uniq` output on Mac vs. Linux. It'll be an easy fix, I just need to see what your data/reuse_subsample_\* files look like.

**Comment by hlapp** [comment]
I've put the two `data/reuse_subsample_*` files on Dropbox:
https://www.dropbox.com/sh/n71cb19tag2bzrz/v-PoxtQ3V9

And yes, I am on a Mac.

**Comment by hlapp** [comment]
Fixed by #8
